### PR TITLE
Remove installer registry service

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
- Remove InstallerRegistryService

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/14153

### CODE REVIEW

**Comment by mkuznyetsov** [comment]
ci-test

**Comment by mkuznyetsov** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
@mkuznyetsov : there was outage on CRW CI https://github.com/eclipse/che/pull/14193#issuecomment-520325980
I have restarted E2E happy path tests again.

**Review by amisevsk — APPROVED** [review]

**Comment by amisevsk** [comment]
I'm seeing test failures in `LegacyEnvVarEnvironmentProvisionerTest.java`, it seems like machineConfigs still contain installers.

**Comment by mkuznyetsov** [comment]
ci-test

**Comment by l0rd** [comment]
Removing label `do-not-merge/hold`. @mkuznyetsov can you rebase please?

File: wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.2.0/1__remove_installers.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+DROP TABLE installer_serverconf_attributes;
+DROP TABLE installer_dependencies;
+DROP TABLE installer_servers;
+DROP TABLE installer_properties;
+DROP TABLE installer;
```
Comment (sleshchenko) [inline wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.2.0/1__remove_installers.sql]: Please, add a new line

**Review by sleshchenko — COMMENTED** [review]
Please consider also removing the following classes InstallerStatusJsonRpcMessenger, InstallerConfigProvisioner, InstallerLogEvent

**Review by skabashnyuk — APPROVED** [review]

**Review by sleshchenko — COMMENTED** [review]
`che.infra.kubernetes.*` properties in `che.properties` should be cleaned up as well.
And from helm chart https://github.com/eclipse/che/blob/40ee503a054b60c7b6dc05928d6c0f8d791b2745/deploy/kubernetes/helm/che/templates/configmap.yaml#L23

There are still some classes which should go away like AbstractBootstrapper... AFAIU it would be easier to remove it along with docker infra - Am I right? If yes - please add a one-line comment into removing docker infra issue about it, just not forget to remove it as well.

**Review by sleshchenko — APPROVED** [review]
My comments should not block your PR and can be handled in another PR

**Pull request merged**